index.html

```diff
@@ -788,38 +788,15 @@ <h2 class="icon icon-more">More &hellip;</h2>
 
   <div id="footer">
 
-    <p>Copyright &copy; 2021 The Apache Software Foundation &mdash; Licensed under the <a href="http://www.apache.org/licenses/LICENSE-2.0">Apache License 2.0</a><br>
-      Apache CouchDB, CouchDB, and the project logo are <a href="http://www.apache.org/foundation/marks/list/">trademarks</a> of The Apache Software Foundation</p>
+    <p>Copyright &copy; 2022 The Apache Software Foundation &ndash; Licensed under the <a href="https://www.apache.org/licenses/LICENSE-2.0">Apache License 2.0</a><br>
```

Comment:
this is not where ndashes are usually used, why the change?

Reply:
Doesn't look that wide.

Reply:
not a good enough reason :D — I chose the mash for reasons of typographic conventions.

Reply:
I forgot, we like wider dashes. :D